packages/table/components/SortableHeaderCell.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { cx } from "emotion";
+import { default as HeaderCell } from "./HeaderCell";
+import {
+  sortableHeaderCSS,
+  sortableHeaderIconBaseCSS,
+  styleArrowDirection
+} from "../style";
+
+type SortDirection = "ASC" | "DESC" | null;
+interface IProps {
+  sortHandler: () => void;
+  sortDirection: SortDirection;
+  columnContent: string | React.ReactNode;
+}
+interface IState {
+  hovered: boolean;
+}
+
+export class SortableHeaderCell extends React.Component<IProps, IState> {
```

Comment:
We have a similar component called `HeaderCell`, so I think it makes sense to change the original name, `SortableColumnHeader`. Happy to change back if anybody feels `SortableColumnHeader` is clearer